spec/lucky/params_spec.cr

```diff
@@ -443,6 +443,109 @@ describe Lucky::Params do
     end
   end
 
+  describe "nested_arrays" do
+    it "gets nested arrays from form encoded params" do
+      request = build_request body: "user:name=paul&user:langs[]=ruby&user:langs[]=elixir",
+        content_type: "application/x-www-form-urlencoded"
+
+      params = Lucky::Params.new(request)
+
+      params.nested_arrays?(:user).should eq({"langs" => ["ruby", "elixir"]})
```

Comment:
If this is `eq` not `includes` shouldnt `"name" => "paul"` be in the params hash?

Reply:
This one was a little tricky. The method only returns the nested arrays, it doesn't return any other keys. This is similar to how the file param stuff works.

The reason why is because I either had to turn the non-array pairs in to `"name" => ["paul"]` to keep the single  `Hash(String, Array(String))` type, or, have the value be a union in which case calling methods on the values becomes a little more difficult. If we merged all of them in to a single method, then the type signature becomes pretty gnarly.